contrib/lib/src/space_helmet/policy.rs

```diff
@@ -0,0 +1,187 @@
+use std::borrow::Cow;
+use rocket::http::uri::Uri;
+use rocket::http::Header;
+
+
+/// [Referrer-Policy](
```

Comment:
It would be nice to:

  1. Say something about what this does, instead of just a link.
  2. Indicate when a user might want to use this.
  3. Annotate each variant, suggesting its purpose.

Same applies to every other policy structure here.